Tests/Functional/Service/SelectionBuilder/ContextMenuSelectionServiceTest.php

```
@@ -61,6 +61,10 @@ public function generateSelectionReturnsContextMenuArrayForPageWithStatus(): voi
         self::assertArrayHasKey('assignee', $result);
         self::assertArrayHasKey('comments', $result);
         self::assertSame('change', $result['1']['callbackAction']);
+        // color/icon are exposed so StatusItemProvider can pass them to context-menu-actions.js
+        // for the optimistic status-header update.
+        self::assertSame('blue', $result['1']['color']);
```

PHPStan (2.2.8)

[error] Offset '1' does not exist on array<string, mixed>.


(offsetAccess.notFound)